proto/pocket/migration/event.proto

```diff
@@ -68,13 +68,21 @@ message EventMorseSupplierClaimed {
   // The unstaked balance which was claimed.
   cosmos.base.v1beta1.Coin claimed_balance = 2 [(gogoproto.jsontag) = "claimed_balance", (gogoproto.nullable) = false];
 
+  // TODO_IN_THIS_COMMIT: update...
   // The hex-encoded address of the Morse account whose balance will be claimed.
-  string morse_src_address = 3 [(gogoproto.jsontag) = "morse_src_address"];
+  string morse_operator_address = 3 [(gogoproto.jsontag) = "morse_operator_address"];
+
+  // TODO_IN_THIS_COMMIT: update...
+  // TODO_IN_THIS_COMMIT: use in msg handler!
+  string morse_singer_address = 6 [(gogoproto.jsontag) = "morse_singer_address"];
```

Comment:
Wdyt of having:

- `string morse_address` // (i.e. operator)
- `strng morse_output_address` // (i.e. owner)
- `bool is_output_signer`

Trying to be "clearer". Feel free to reincoporate "operator" in the field naming though!

Reply:
My thinking is:
- `morse_output_address` would be more consistent with Morse nomenclature 👍
- this (:point_up_2:) would allow `morse_output_address` to be optional 👍
- `morse_address` does not sufficiently disambiguate 👎
- `morse_operator_address` seems less consistent ... **how about `morse_node_address` or `morse_servicer_address`**? ❓